mslice/scripting/helperfunctions.py

```diff
@@ -122,13 +124,20 @@ def add_cut_plot_statements(script_lines, plot_handler, ax):
     add_plot_options(script_lines, plot_handler)
 
     if plot_handler.is_changed("x_log"):
-        script_lines.append("ax.set_xscale('symlog', linthreshx=pow(10, np.floor(np.log10({}))))\n".format(
-            plot_handler.x_axis_min))
+        if LooseVersion(mpl_version) < LooseVersion('3.3'):
+            script_lines.append("ax.set_xscale('symlog', linthreshx=pow(10, np.floor(np.log10({}))))\n".format(
+                plot_handler.x_axis_min))
+        else:
+            script_lines.append("ax.set_xscale('symlog', linthresh=pow(10, np.floor(np.log10({}))))\n".format(
+                plot_handler.x_axis_min))
 
     if plot_handler.is_changed("y_log"):
-        script_lines.append("ax.set_yscale('symlog', linthreshy=pow(10, np.floor(np.log10({}))))\n".format(
-            plot_handler.y_axis_min))
-
+        if LooseVersion(mpl_version) < LooseVersion('3.3'):
+            script_lines.append("ax.set_yscale('symlog', linthreshy=pow(10, np.floor(np.log10({}))))\n".format(
+                plot_handler.y_axis_min))
+        else:
+            script_lines.append("ax.set_yscale('symlog', linthresh=pow(10, np.floor(np.log10({}))))\n".format(
```

Comment:
This is picky, but it would be good to modernise and use f-strings instead of the format function. Same for in the test file as well